src/api.jl

```diff
@@ -161,13 +182,14 @@ function infer(fieldids::Vector{Tuple{Int, Int, Int}},
 
     # If there are no objects of interest, return early.
     if length(target_sources) == 0
-        return Dict{Int, Dict}()
+        return Dict{Int, Dict}(), (0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0)
```

Comment:
Well, IMO you're putting things in ```bin/celeste.jl``` that should be in ```api.jl``` (and tested), and there are things in ```api.jl``` that should be in ```SDSSIO.jl``` (or something like it).  Again, IMO, timing measures are attributes of an API and should be defined with the API in its own module, not in a binary script.

After the paper I think we should organize the whole thing better.  In the meantime, just go ahead and do what you think is best that gets you up and running the fastest that doesn't interfere with the rest of us by moving stuff around too much, and we'll be fine.  :)